zulipterminal/ui_tools/boxes.py

```diff
@@ -1112,26 +1112,39 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         if not reactions:
             return ""
         try:
+            my_user_id = self.model.user_id
             reaction_stats = defaultdict(set)
             for reaction in reactions:
                 user_id = int(reaction["user"].get("id", -1))
                 if user_id == -1:
                     user_id = int(reaction["user"]["user_id"])
-                reaction_stats[reaction["emoji_name"]].add(user_id)
+                user_name = reaction["user"]["full_name"]
+                if user_id == my_user_id:
+                    user_name = "You"
+                reaction_stats[reaction["emoji_name"]].add((user_id, user_name))
 
             sorted_stats = sorted(
-                (reaction, count) for reaction, count in reaction_stats.items()
+                (reaction, list(count)) for reaction, count in reaction_stats.items()
             )
 
-            my_user_id = self.model.user_id
+            for stats in sorted_stats:
+                stats[1].sort(key=lambda id: id[0])
```

Comment:
Whats the need of sorting based on id's here? Does that mean users who have a smaller id are preferred? From [this](https://chat.zulip.org/#narrow/stream/277-discussions/topic/Next.20discussion.20topic/near/1319047) message, it looks like your ordering and webapps ordering differ.

Reply:
- Every time ``reacting user_ids`` are fetched, I noticed that ordering was even if there was no change in ``reacting users`` for a reaction. Therefore, I sorted the ``ids``.
- Next, in ``test_reaction_views`` because of random ordering, tests were failing.